documentation/source/user_section/tutorials/spinalcord_segmentation.rst

```diff
@@ -0,0 +1,243 @@
+
+.. warning::
+
+   This tutorial uses sample MRI images that must be retrieved beforehand. Please download and unzip `sct_course_london20.zip <https://osf.io/bze7v/?action=download>`_ , then open up the unzipped folder in your terminal and verify its contents.
+
+   .. code:: sh
+
+      $ ls
+      multi_subject single_subject
+
+   We will be using images from the ``single_subject/data`` directory, so navigate there and verify that it contains subdirectories for various MRI image contrasts.
+
+   .. code:: sh
+
+      $ cd single_subject/data
+      $ ls
+      dmri  fmri  LICENSE.txt  mt  README.txt  t1  t2  t2s
+
+.. note::
+
+   Both ``sct_propseg`` and ``sct_deepseg_sc`` are designed to work with four common MRI image contrasts: T1 weighted, T2 weighted, T2* weighted, and DWI. If your image data uses a contrast not listed here, select the closest visual match to the available options. For example, FMRI images have bright cerebrospinal fluid (CSF) regions and dark spinal cord regions, so the T2 contrast option would be an appropriate choice.
+
+   .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/image_contrasts.png
+      :align: center
+      :figwidth: 75%
+
+      Image contrasts and their corresponding ``-c`` settings
+
+Algorithm #1: ``sct_propseg``
+*****************************
+
+SCT provides two command-line scripts for segmenting the spinal cord. The first of these is called ``sct_propseg``. This tutorial will explain the how the script works from a high-level theoretical perspective, and then it will provide two usage examples.
+
+Theory
+------
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/optic_steps.png
+   :align: right
+   :figwidth: 20%
+
+   Centerline detection using OptiC
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/mesh_propagation.png
+   :align: right
+   :figwidth: 20%
+
+   3D mesh propagation using PropSeg
+
+``sct_propseg`` itself is a single command, but internally it uses three processing steps to segment the spinal cord.
+
+#. Detect the approximate center of the spinal cord automatically using a machine learning-based method (OptiC). This is an initialization step for the core algorithm, PropSeg.
+#. Create a coarse 3D mesh by propagating along the spinal cord (PropSeg).
+#. Refine the surface of the mesh using small adjustments.
+
+   .. note::
+
+      The centerline detection step is also provided in a standalone script called ``sct_get_centerline``.
+
+Example: T2w image
+------------------
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t2_image.png
+  :align: right
+  :figwidth: 10%
+
+  t2.nii.gz
+
+From the ``single_subject/data`` folder, navigate to the ``t2`` directory, and verify that it contains a single T2-weighted image:
+
+.. code:: sh
+
+   $ cd t2
```

Comment:
I know it's common to display "$" to show that this is what users need to write, but when doing "quick" copy/paste by double clicking on the line, it includes the "$" and then we need to manually remove it, which is cumbersome. Also I anticipate that some users will include the "$", have an error and won't understand the source of the error.

Suggestion:
- get rid of "$"
- find a way to display "$" "outside" of the copy/pastable field. I've seen some webpage where this was possible. Maybe sphinx has some tools to do that...?